test/cache.test.ts

```diff
@@ -44,6 +47,7 @@ describe("cache", () => {
                 bucket: "sdm-th3by4d5-sweetheart-of-the-rodeo-goal-cache",
                 enabled: false,
                 path: "goal-cache",
+                region: "us-east2",
```

Comment:
Perhaps put this in the next test since this test is testing the default config?